tasks/bump.rake

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require 'bump'
+
+namespace :bump do
+  desc 'Bump to a pre-release version'
+  task :pre do
+    bump_and_push('pre')
+  end
+
+  desc 'Bump to a patch release version'
+  task :patch do
+    bump_and_push('patch')
+  end
+
+  desc 'Bump to a minor release version'
+  task :minor do
+    bump_and_push('minor')
+  end
+
+  desc 'Bump to a major release version'
+  task :major do
+    bump_and_push('major')
+  end
+
+  desc 'Bump to a manually-specified version (in x.x.x... form; do not add initial "v")'
+  task :manual, [:ver] do |_t, args|
+    bump_and_push(args[:ver])
+  end
+end
+
+def bump_and_push(version)
+  sh %(gem bump --version #{version})
+  update_changelog
+  sh %(bundle list > /dev/null)
```

Comment:
I meant to add a comment to describe it, and maybe there's a way around it.  This basically causes `bundle` to be run, which updates the `Gemfile.lock` file.  I found that if I didn't do this step, for some reason when doing the `gem tag --push` step below, it complains that there is an uncommitted change to `Gemfile.lock`.  However, if I don't do this line here, the `Gemfile.lock` file isn't changed yet when doing the git add/commit, so something with the `gem tag --push` is causing it to update at that point, and for some reason the `gem bump --version xxxx` above does NOT update the `Gemfile.lock` file.

I hope that makes sense.  I admit, this does feel like kind of a kludgy workaround, but having it here was the only way I could get it to run properly without errors.

Reply:
According to the docs, `bundle` is supposed to run automatically as part of the bump step (there's an option to _not_ run `bundle`). Maybe somehow the command line version isn't automatically running the post-bump bundle step?

Reply:
"I found that if I didn't do this step, for some reason when doing the gem tag --push step below, it complains that there is an uncommitted change to Gemfile.lock." It sounds like in fact the `Gemfile.lock` file **is** changed? Perhaps it looks like it wasn't updated because it was added to the commit but the commit didn't happen yet.

Reply:
Without the `bundle list` step, the Gemfile.lock file doesn't get changed until somewhere INSIDE the `gem tag --push` step itself.  When I was testing this, I was having it output `git status` after every line, and the lock file never showed up, yet the `gem tag --push` step still failed saying Gemfile.lock was not committed (and then after the failure, `git status` would show it).  Maybe it's a bug with `gem tag --push` or `gem bump`?  I'm not sure.

That said, the Bump gem seems to be handling things better so I'll leverage that.